lib.nix

```diff
@@ -51,26 +51,29 @@ in lib // (rec {
   andreasKey = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC+jXbzao695RgcORzsfMZIUjaE802hTv3YkN4oRQUa5Ucninw/ngaPMI0Ym8IKCW7N9B40lOOZuIiGv2zEL6Y55/vUS7Y9hBX4vlB0qLC75+rj80JHyc8zEMBPNNrR3j+EXvWadCx0EYIEV9DsqkLNrUOtweqkPOfA4IUypX1xVmaPpUsqXTThjy8z4jJPgFhN6wVDRQ66WNzMjFwt1B9X11jNUy0WLP2idzqLoko/gkZiNOslJVWniOzeHiPrHOmiEWEj1xSaZ05r1FeJR/UKYuKQQ1sZEoqeXM4BgRDjqpJN/+AtCIC8CTltRu3WVMfpiwma5BAA2Nbbe5rc8xACgZvzJLtADJLCyyhRzVeOhW0c9t9RcG1zafaLJ81VBSrpPog0pcT4C/CHmgveti0OAY7uNATsRq6Nm6fGaXLzsRdOjtytS7JEVb11ZMUbDLTlj/uu0RSzrtmDdLbsS1X/7BHrnd9Q4XKS0RJVLjDmi4htdLSZF1675g7ZhZGSu3sMP6IGUIsE0QrPSKuybWBWnkJqP5ytqbxMolK0AziIbgjKOE1DBR24IBQaFhhGPjAErPy2wGLpMdc6Dfnq481d7xzR4Cxz2fqU7P5cISZklHc86F6K/gqdR+ucSFdHl7lPU3jmxGe1pU1+AznCDJ8g1ZOBeLGyJUbNI6kVPoWTCw== andreas_triantafyllos@hotmail.com";
-  samKey = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC5PPVdfBhmWxWYjSosuyjMdIYNjYC/ekz+Whv27wrFNHqxeGgKbXslUTwZX0r+zu/nlJnX9nj3zdVV9LosBB8JF9tfJGui9aBfPuxoIq9SMFSdcpZ8aOh4ZITv7zbsRHMECE8q7D5/a+7UZyTy8pv9g5SuCerHh3m//NIbo08OS9rt8SjqVio+B+rseLF960U3U3wTCtOA+VauTuE4kZfSfmQlEYUjaN3qwp4s5jpO7pgnGxshuqayRyuwJfRa/RYWB5ouSjyxTuo33K42EqT4XFoURkj7evJB5SRR7pm4vJCx4VkclIVmpLIcBiyWje+60zyKhAZEQVqKXedkuQ9748wZl07C6Czs4QiloGAjXv/tRm9YSdoeG5JhskEA8z2SCEQARJGquPH+f5vBltHeVC5K5LW94gSP9bfVBitcCgONVxUguCu0PmJUYKcVVjRi3KtJJzDSTDCjjN3e/mszrZY921yvVEkb7mFATBiHeSdrt55gKcG1vfTToLALIJJFQpGCwAMYUjKEcgq4PZa1UdCY/ynvynLds3mge4Y/X3EnLFsJaepfgNyPnnPg67kEda8uRSDYT8LaoqJpDzc7RQeY4BOfJfAxa8qMDHmp4W+dxHqrMphbH66fwUJAx1MWV8AoPFW0TGrDb3AnHBgoRt/5Fnz2ymy92Wb1KAIt3w== sam@optina";
   alexandersKey = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDUGnVIJ5w4OP9RaTZKM929Wt69UyvbBLyK65FNAkNXraKJ4t7rp41AaiY17byBET5vHxiF7CdVxi2nng6SQKp8VOlCPvF/AN4ZjruFDIXunJLERzf7GCR+bQPu4E/uj1hR9lIs8/jRKtHN6cZ7jcRni8b7BSIISInHTDwGm7XhdpRQlrxMK67oMXQuuOnEEShmH+ER0ICGk/o33SIAK2bebfRgCdI7N8iMZ/vw9ACHptN1wG70H2OVZADFY7J2mS0EFz+iFx5aOn/EbQfgNb8y9+K4+kfWxl/lwFw0Lj6YwUMEvqlqtHRYKq7uLT18JzYfM4mtGkYaNGDmzfqqpq5UTIW0txr9luNf5CRZeJctsY/OpxV1dPZXIMMENl1Y3a2GnD2YfKZ04eQrUgFN4R5J9S3dhkqUNlzc72/dkbCbWVk86+uG4k+DAm5MCua9xaF7jEuwd/BnCw72ce06hf5soyiIpOLhkuMyTY6/rXiSYETdGlJKTgx1S3bqa5FD4yXA++fwxoOLww/ZMRRdiufSk3KJBvAQMP0lKxR7uQA9S4ZGzLifWII21f2x1mLvknzzhKY4hZLmXA/TIeeygWYmIF5ldoTVtWP6duLspFzC3Fei+tm6ZzYcuuAX6jaT76Kkrsy9bQOjuI/mwwILzD6qZeu41Y/FsvQzgVqJ91HX/w== codieplusplus@apax.net";
 
-  devKeys = [ domenKey alanKey jakeKey dshevchenkoKey larsKey philippKey kosergeKey ksaric sectore michaelKey akegaljKey alfredoKey rodneyKey vasilisKey andreasKey samKey alexandersKey ];
+  devKeys = devOpsKeys ++ [ alanKey dshevchenkoKey larsKey philippKey ksaric akegaljKey alfredoKey vasilisKey andreasKey alexandersKey ];
```

Comment:
i think put list items one per line and sort them.
We previously had merge conflicts on these lines which were more difficult than necessary to resolve.

Reply:
yeah, did that actually : -)